src/pudl/metadata/fields.py

```diff
@@ -741,6 +741,10 @@
         "type": "string",
         "description": "Generator ID is usually numeric, but sometimes includes letters. Make sure you treat it as a string!",
     },
+    "generator_id_epa": {
```

Comment:
Ugh, we should really switch to `generator_id_eia` as well. I wonder how many search-and-replaces it would take.

Reply:
I knowwww, also `boiler_id`. It's on my list of things to do once I fix everything else...

Reply:
> once I fix everything else...

🤣🤣🤣